src/Security/PersonTokenAuthenticator.php

```diff
@@ -144,4 +118,43 @@ public function onAuthenticationFailure(Request $request, AuthenticationExceptio
             403
         );
     }
+
+    /**
+     * Authentication success.
+     *
+     * @param Request        $request     A Symfony Request, req by interface.
+     * @param TokenInterface $token       A Symfony user token, req by interface.
+     * @param string         $providerKey The name of the used firewall key.
+     *
+     * @return Response The response or null to continue request.
+     */
+    // Next line to be ignored because implemented function does not have type-hint on $providerKey.
+    // phpcs:ignore
+    public function onAuthenticationSuccess(Request $request, TokenInterface $token, $providerKey)
```

Comment:
Does it break if it is type "string"?

Reply:
The prototype will fail, I think it's fixed in Symfony 5.